src/hooks/useToast.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+
+import ToastMessage from "../components/common/ToastMessage";
+
+export default function useToast(initialState = false) {
+  const [sendToast, setSendToast] = useState(initialState);
+
+  const handleSendToast = () => setSendToast(true);
```

Comment:
미리 정한 컨벤션에 따라, 한 줄이지만 아래처럼 해주는 게 어떨까요?
```suggestion
  const handleSendToast = () => {
    return setSendToast(true)
  };
```

Reply:
앗 이부분 지난번에 양욱님과 한번 이야기 나눈적 있는데 80자 안넘어서 한줄로 쓰긴했습니다! 모달과 컨벤션을 맞추려구요 ㅎㅎ 괜찮을까요??

혹 바꿔야하면 수정될 부분들이 좀 더 늘어날 것 같긴 합니다..!

Reply:
아네넵! 간결하게 쓰는 것도 좋죠! 저도 고려하면서 작업하겠습니다~